libs/blocks/tabs/tabs.css

```diff
@@ -423,7 +423,7 @@
   inline-size: var(--grid-container-width);
 }
 
-.tabs.radio div[role="tablist"] .tab-list-container {
+.tabs.radio div[role="tablist"].tabList .tab-list-container {
```

Comment:
Wouldn't this also do the trick? Same comment for L508
```suggestion
.tabs.radio .tabList[role="tablist"] .tab-list-container {
```

Reply:
<img width="1711" alt="Screenshot 2025-02-10 at 11 33 20" src="https://github.com/user-attachments/assets/4af4bb6f-893e-4b8f-afc3-5e4e8f4ac3d2" />

Need more specificity because `width: auto` messes up container width